.github/workflows/tests.yml

```diff
@@ -0,0 +1,65 @@
+name: Test Suite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+    test:
+        runs-on: ${{matrix.operating-system}}
+        strategy:
+            matrix:
+                operating-system: [ ubuntu-latest ]
+                php: [ "5.6", "7.0", "7.1", "7.2", "7.3", "7.4" ]
+        name: PHP ${{matrix.php }} Unit Test
+        steps:
+            - uses: actions/checkout@v2
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                php-version: ${{ matrix.php }}
+            - name: Install Dependencies
+              uses: nick-invision/retry@v1
+              with:
+                timeout_minutes: 10
+                max_attempts: 3
+                command: composer install
+            - name: Run Script
+              run: vendor/bin/phpunit
+
+    test_php55:
```

Comment:
Maybe I'm blind, but is there any difference between this and the 5.4 test and the regular one? I remember there used to be with the old php setup action, but I can't see any now.

Reply:
No there isn't. I assumed that the matrix above didn't support 5.5 so we had to create a special one.

Reply:
(updated)